packages/wdio-allure-reporter/src/index.js

```diff
@@ -227,7 +227,7 @@ class AllureReporter extends WDIOReporter {
         }
 
         // add beforeEach / afterEach hook as step to test
-        if (this.options.disableMochaHooks && isMochaEachHooks(hook.title)) {
+        if (isMochaEachHooks(hook.title)) {
             if (this.allure.getCurrentTest()) {
                 this.allure.startStep(hook.title)
```

Comment:
I have checked and before the behaviour was like this:
```
    if (isMochaEachHooks(hook.title)) {
      this.allure.startStep(hook.title)
      this.setCaseParameters(hook.cid);
      return
    }
```
I assume this behaviour is still needed (the setCaseParameters)

Reply:
I tried to look into commit history and couldn't find reference to it. For those lines in onHookStart, I only reverted in back to before #4773. If the setCaseParameters is needed, I can try adding it in and test again.

Reply:
@erwinheitzman I tried adding the setCaseParameters line in, and it didn't look like the hook.cid was used in the report for the mocha each hooks.

![Screen Shot 2020-02-19 at 4 13 13 PM](https://user-images.githubusercontent.com/13144677/74888600-45efc380-5333-11ea-8242-a100515221e0.png)